keps/sig-instrumentation/647-apiserver-tracing/README.md

```diff
@@ -170,8 +170,8 @@ expected from the request.
 
 Alpha
 
-- [] Implement tracing of incoming and outgoing http/grpc requests in the kube-apiserver
-- [] Integration testing of tracing
+- [X] Implement tracing of incoming and outgoing http/grpc requests in the kube-apiserver
+- [X] Integration testing of tracing
```

Comment:
Sorry - can't comment on the lines where I would like to, so commenting here:

L212: have those been added? if so, can you link them?

L300: can this be somehow configured? If not, what is the default cache size (i.e. what order of magnitude increase I can expect?)

Reply:
Default cache size is 2048 spans, with each span being between 850 bytes, and 3000 bytes, depending on how many attributes we attach to the span.  So a full cache (which might happen if it is unable to export spans, but is still collecting them) is ~5 Mb.  This can also be configured when we set up the SDK (i.e. in code, not via flags right now).

Reply:
Added a link to tests that enable the feature gate